tests/modules/suite2p_registration/test_suite2p_registration_utils.py

```diff
@@ -164,3 +165,34 @@ def test_check_movie_against_raw():
         check_movie_against_raw(corr_data=corr_data[rand_index],
                                 raw_hdf5=h5_file_loc,
                                 h5py_key=h5_key)
+
+
+def test_check_and_warn_on_datatype():
+    """Test that warnings are thrown when the wrong types data types are
+    specified.
+    """
+    h5_file = tempfile.NamedTemporaryFile('w', suffix='.db')
```

Comment:
Obviously it works, but: why not `suffix='.h5'`?

Reply:
Copy paste job. Just missed it.